static/build/miyoo/app/lang/gl.lang

```diff
@@ -0,0 +1,161 @@
+{
+	"lang": "Galego (Galician)",
+	"0": "Experto",
+	"1": "Favoritos",
+	"2": "Sistemas",
+	"3": "ARCADE",
+	"4": "NEOGEO",
+	"5": "FC",
+	"6": "SFC",
+	"7": "GB",
+	"8": "GBA",
+	"9": "MD",
+	"10": "GPSP",
+	"11": "SMS",
+	"12": "PS",
+	"13": "PCE",
+	"14": "OUTROS",
+	"15": "Axustes",
+	"16": "INFO. XOGO",
+	"17": "INFO. CONSOLA",
+	"18": "Recentes",
+	"19": "ARQUIVO",
+	"20": "ADC INFO",
+	"21": "EncDec Test",
+	"22": "Mapeo botóns",
+	"23": "Idioma",
+	"24": "Volume",
+	"25": "Volume música",
+	"26": "Brillo",
+	"27": "Actualizar ROMs",
+	"28": "Restauración de fábrica",
+	"29": "Modo flasheo",
+	"30": "Información dispositivo",
+	"31": "Proba hardware",
+	"32": "Info ADC",
+	"33": "Estado:",
+	"34": "Retardo:",
+	"35": "Taxa OK:",
+	"36": "Taxa fallos:",
+	"37": "DETIDO",
+	"38": "CORRENDO",
+	"39": "ADC:",
+	"40": "Cargando",
+	"41": "OTG",
+	"42": "Desconectado",
+	"43": "Erro",
+	"44": "Favoritos",
+	"45": "Cancelar",
+	"46": "OK",
+	"47": "segundos",
+	"48": "en",
+	"49": "Lista recentes",
+	"50": "Limpar",
+	"51": "Abrir",
+	"52": "Eliminar",
+	"53": "Refrescar",
+	"54": "Refrescar ROMs",
+	"55": "Engadir como favorito",
+	"56": "Actualizar",
+	"57": "¿Actualizar firmware?",
+	"58": "Capacidade SD baixa\n%dM restante",
+	"59": "Número modelo",
+	"60": "Número serie",
+	"61": "Versión",
+	"62": "CPU",
+	"63": "%dMHz",
+	"64": "RAM",
+	"65": "%dM",
+	"66": "Tarxeta SD",
+	"67": "Restaurar todo o sistema",
+	"68": "Restaurar todo o sistema\nO contido SD\nnon se eliminará",
+	"69": "Actualizar ROMs na tarxeta SD",
+	"70": "Restaurar",
+	"71": "por defecto",
+	"72": "rotación90",
+	"73": "map3",
+	"74": "Localización",
+	"75": "CARGANDO",
+	"76": "Cargando",
+	"77": "Limpar lista",
+	"78": "Limpar recentes\nO contido SD\nnon se eliminará",
+	"79": "Limpar favoritos",
+	"80": "Limpar xogos desta lista\nO contido SD\nnon se eliminará",
+	"81": "Eliminar de favoritos",
+	"82": "Eliminar xogo desta lista\nO contido SD\nnon se eliminará",
+	"83": "Escanear xogos",
+	"84": "INFO. XOGO",
+	"85": "Crear caché",
+	"86": "Apagar",
+	"87": "Apagar en %d segundos",
+	"88": "ABRIR",
+	"89": "VOLTAR",
+	"90": "NAVEGAR",
+	"91": "MENÚ",
+	"92": "Continuar",
+	"93": "Gardar",
+	"94": "Cargar",
+	"95": "Volume",
+	"96": "Saír do xogo",
+	"97": "Gravar %d/%d",
+	"98": "Baleiro",
+	"99": "Sen imaxe",
+	"100": "Carga fallida",
+	"101": "Gardado fallido",
+	"102": "Ranura %d gardada",
+	"103": "Pantalla completa",
+	"104": "Acender",
+	"105": "Apagar",
+	"106": "Menú nativo",
+	"107": "Aplicación",
+	"108": "¿Apagar o dispositivo?",
```

Comment:
~~¿Apagar o dispositivo?~~
**Apagar o dispositivo?**

There is no inverted question mark in Galician.